### DISCUSSION TITLE
Documenting my new Nixie build...

### DISCUSSION THREAD

**Discussion by chadhobson** [tagyoureit/nodejs-poolController#484] [Nixie]
Rather than keep all this in my other semi-unrelated topic, I'm going to document this controller build here in this thread for someone in the future.

I am following the "dream" single body system located [here](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller). Anyone who knows me, knows I'm opinionated and lack a filter, so this is exactly that. My opinion, unfettered.

_*** I have paid for every single part of this system and nothing is sponsored! And the prices on that link are so far away from reality now it's not even funny but we all know who's fault that is. #FJB_

So first off, my pool was stupid simple when I started this, so let's see where we end up. Its a 17,500 gallon inground vinyl liner pool. Something like 14 ft wide by 32 ft long. Oval in the back, kidney in the from where the stairs come in. Its getting a ne wliner next spring and will be getting new LED lights then as well.

At the start, I had a Mustang 1HP single speed pump, Hayward sand filter, AutoPilot SWG, AquaCal Heatwave heater, and half-burnt fiberglass lights. This pool was built in 2008 and literally everything is "of the time". And thats no good, so this summer I'm redoing most of it. 

Luckily, other than the pump, all my equipment works.... to the tune of about $170/month increase in my electric bill when I open the pool. The SWG was replaced about five years ago along with the sand filter, so it's not ALL old equipment, but it's certainly not energy efficient or, ahem, smart. But I digress...

The first decision I made was that everything being replaced would be replaced with Pentair equipment, if possible. They seem to be the Cadillac of the pool world and the prices certainly reflect it. YMMV.

So this weekend, I got the first task done. I replaced the 1HP Mustang with a 1HP Pentair i1 Intellisense pump.  Fun fact: pipe dope works GREAT on the suction side. Not so much on the return side. So I lost a day rebuilding that particular pipe from the pump to the filter (twice) but now with the use of teflon tape instead of dope, we're leak free, air tight, and wow... the old pump ran at 3450 RPM and created 12 PSI at the filter. This new pump at the same RPM is creating 28 PSI at the filter!

Why is that important? Because with the new pump, I only have to run at something around 1800 RPM to achieve 12 PSI at the filter. This thing is definitely efficient! Best part, I pretty much can't hear the pump at all at anything under 2000 RPM which is taking some getting used to. It reminds me of when I redid the sump pump plumbing in my basement to be quieter and then when I couldn't hear it kick on during a storm, I freaked out. Still do, in fact.

So now the stage is set for the next part which will be an IntelliChlor IC40 (where's my thread tape!?!) and the Nixie. I went with a larger SWG so it could run less and, hopefully, last longer. Time will tell.

For the Nixie, again I went with the "dream" build, so I've got pretty much all the parts from the link above. The new fiberglass enclosure in this system is way too large for my setup (see pic below) but I don't mind it as it gives me room to grow into some lighting stuff I want to do and I can always take it with me to the next house.

As of now, I am waiting on the Pentair 3HP relay after mistakenly ordering a 24VAC pump relay instead of DC. Probably won't be the last time I have to return something, so it is what it is. I also need to figure out how to mount this hulking box on a 4x4 fence post.

Anyway, I'll leave this post with a photo of the current state of my Nixie build. If any pros see anything off, please let me know. You won't hurt my feelings at all.

In this photo, I have the vented enclosure which required a trip to Lowes for longer screws because this company obviously doesn't do any QA before they ship a fiberglass product which is by FAR the most aggravating part of this project so far. I won't go into that one though. If I could do it all over again and had the time to research, I would have 100% picked another brand.

As of right now, I have all the hardware in the box that I need. I'm waiting on some 2 pole jumpers for the terminals so I can wire it up to a dishwasher cord and finish getting the automation hat and relay hat configured without doing it "live", but other than that the Pi is running nodeJS-poolController and dashPanel and is working as well as could be expected given nothing is hooked up to it.

I'll keep this post updated as I work through this. Thanks to @rstrouse for pushing me off the ledge! ;-)

![IMG_3052](https://user-images.githubusercontent.com/1839161/173259066-afbc9d78-e417-451d-91f6-c950b28059fe.jpeg)

**Comment by rstrouse** [comment]
Awesome ready to help when we can.  I thought about updating the cost sheet but folks probably need every reminder they can get.

**Reply by chadhobson** [reply]
Amen to that. Any thoughts on best way to mount this monster box to a 4x4 fence post? I know it's not ideal but its where the AutoPilot has been mounted for 15 years and it's fine, so thats good enough for me. I was thinking drill two holes in the back of the box then throw some washers on some deck screws and send it. Any better idea than that?

**Comment by rock-crusher** [comment]
I like that you’re documenting this build. It’s fun to read along and see the different ways how people choose to assemble their enclosures. Looks like you’re off to a great start, you won’t regret the size of the box you have as they seem to grow smaller as the wiring begins.
Thanks again for taking the time to share with all of us and don’t forget to keep the pictures coming!
Oh, one question please, just curious as to why the 48v ??

**Reply by chadhobson** [reply]
The automation hat for the Pi is a proper piece of building automation hardware and that stuff all runs on 48V. Another added benefit being that the hat also provides 5V to power the Pi so one less power supply in the box.

So, between the controller and the pump/SWG combo, will be a 3HP relay that is opened via a 48 VDC current, so thats the real reason. And, hence, why the 24VAC relay won't work for this setup. We've got to have DC inside the controller box with the exception of the 110V AC line that powers that power supply (terminal blocks are the white/black/green in the top right of pic).

@rstrouse I got the 3HP relay today. I see two "mounting holes" on it but I'm guessing I'm just gonna throw it in a generic outdoor junction box from Lowes, so how did you mount yours? I was thinking 3M two sided tape as it doesn't seem to weigh much at all but since this is the box with the actual high voltage in it, I'm doubting that decision. Pic would be awesome if you have one?

**Reply by rock-crusher** [reply]
> The automation hat for the Pi is a proper piece of building automation hardware and that stuff all runs on 48V. Another added benefit being that the hat also provides 5V to power the Pi so one less power supply in the box.
> 
> So, between the controller and the pump/SWG combo, will be a 3HP relay that is opened via a 48 VDC current, so thats the real reason. And, hence, why the 24VAC relay won't work for this setup. We've got to have DC inside the controller box with the exception of the 110V AC line that powers that power supply (terminal blocks are the white/black/green in the top right of pic).
> 
For some reason this comes to mind
![Up in Smoke](https://user-images.githubusercontent.com/94593954/173682431-e88a4a97-20de-434a-a511-73b01ca3b06f.jpg)

**Comment by chadhobson** [comment]
Did some research today and discovered my AquaCal Heatwave does indeed have RS-485 connection despite being a million years old. 

@rstrouse Sorry to keep @'ing you but I saw your post in an issue someone posted about the Aquacal protocol sometimes matching Pentair and sometimes not. Should I bother trying to wire this RS-485 connection into the bus or just leave it separate? I honestly don't use the heater much (too much $$$ and not enough heat) but I'm already this deep into it so...

**Reply by rstrouse** [reply]
I think it is the same as the UltraTemp protocol but there will be no difference with either method.

**Comment by rstrouse** [comment]
Hey I hope you aren’t putting 48v into the MEGA-BAS.  It is rated at 24v ac or dc.

**Comment by chadhobson** [comment]
Oh shit. You’re both right. Nice catch!

Why did I get 48V in my head while ordering parts? Grrr… another return.

I can’t get to the software part of this project soon enough.

**Comment by chadhobson** [comment]
Got the correct power supply on the way now. Thanks again for catching that.

I'm still hung up on this relay. From the wiki it says...

_If you have a variable speed or flow pump that has an RS485 communication port, it should never be connected to a relay and be powered at all times. No relay is required or desired for this scenario. In fact, you will be defeating the circuit protection on the pump that energizes the pump circuitry to ensure cold weather does not harm the pump drive._

That is 100% what I have, so I shouldn't need a relay. But then the next line says...

_Pumps that require a relay typically come in one of three categories. These are Filter Pumps, Feature Pumps, and Booster Pumps. Each pump type has a special purpose within your pool controller. If you are connecting a filter pump this is the main pump for the pool and its primary purpose is to circulate the water for the pool or pool/spa combo. I would recommend that you label this your filter circuit._

Hmmm... well I only have one pump - with no features or boosters - so that makes it the "main pump" and its only purpose is to "circulate water", so.... now I'm confused.

Do I need a relay or not?

**Reply by rstrouse** [reply]
What model is the pump?

**Reply by chadhobson** [reply]
![IMG_3028](https://user-images.githubusercontent.com/1839161/173860868-83e985fd-d170-4505-84b4-b51631f6f57a.jpeg)

1HP Pentair i1 Intellisense
EDIT: I added a pic so I can show off my plumbing skills. I'm painting all the pipe black as I replace it.

**Reply by orrious** [reply]
I do not use a relay with these pumps.  There is always power to the pump, njs-pc controls the pump via rs-485.

**Reply by chadhobson** [reply]
Thats my understanding as well and I do run the pump 24/7 at various different speeds.

**Reply by rstrouse** [reply]
You want power on the pump direct from the circuit breaker and do not want to run a relay in between it.  The second statement is related to pumps that are not RS485 controlled.

**Reply by chadhobson** [reply]
Excellent! Something else I can return. This build is getting cheaper by the minute!

**Comment by rock-crusher** [comment]
Don’t forget though, you still need to control your IC40 via a filter pump relay so it is only energized when the filter pump is running and circulating water.
Read your IC40 instruction manual, it will explain this in detail.
Not sure if the relays supplied with your particular relay hat are capable of the load requirements for doing this, or if you will still need the use of an auxiliary relay such as your 3hp relay. 
That said, double check this before returning the 3hp relay, you may still need it.

**Reply by chadhobson** [reply]
Excellent point, @rock-crusher.

So the IC40 can run with either 110v or 220v. If I run it at 110v, then I can use one of the relays on the [relay hat](https://sequentmicrosystems.com/products/raspberry-pi-relays-stackable-card?pr_prod_strat=description&pr_rec_id=d479e078c&pr_rec_pid=6211488481475&pr_ref_pid=7549185032444&pr_seq=uniform) as they'll go up to 110v.

But if there's some compelling reason to run the chlorinator at 220v, then I'll need to go back to an external relay. So, I get why you'd want to run a pump at 220 vs 110 but is there any reason to run a chlorinator at 220 vs 110?

My life will be much easier if I run it at 110, IMO.

**Reply by chadhobson** [reply]
So that relay hat has an 8 AMP max. From the specs..

_All relays have NO and NC contacts and can switch 120VAC/8A or 24VDC/8A._

I then dug up the Intellichlor 40 manual and it says...

_**IntelliChlor Salt Chlorinator Model IC40**
Input: 90-130 VAC, 50/60 Hz, 220 Watts (2 AMP) or 220-240 VAC, 50/60 Hz, 220 Watts (1 AMP).
Output: 22-39 VDC @ 7.3 AMPS maximum from the Power Center_

If I'm reading that right, it's only drawing max 2 amps at 110v? So,  should be fine to use the relay hat?

**Reply by orrious** [reply]
@rock-crusher interestingly enough I'm getting ready to wire my IC40 in.  I understand the manual say put it on the relay / timer as your pump, but those folks don't control it via rs-485?  They also have flow switches in them to prevent chlorine generation when there is no flow.    Does nodejs-poolcontroller tell the IC40 to shutoff?  I wasn't planning on switching the power input to my IC40, but may need to rethink now.  Thoughts from others?

**Reply by rstrouse** [reply]
Nixie will send the requisite off commands to the Chlorinator.  I have seen lots of pics of runaway current on the cell so I wouldn’t switch the low voltage side of the surge circuit.

**Reply by orrious** [reply]
Would you switch the 110/220v to the Chlorinator or do you leave the Chlorinator powered 24/7 and use rs-485 for control?

**Reply by rock-crusher** [reply]
orrious, as you already know, the manufacture states this in the manual: 
“the IntelliChlor AC power center wires must be connected to the LOAD SIDE OF THE FILTER PUMP RELAY”
And their explanation is this:
“This ensures that the IntelliChlor SCG cell produces chlorine only when the main filter pump is on”
The fact that a filter pump is controlled via RS485 does not change the above statements.
Follow the manufactures instructions and use the filter pump relay.

**Reply by rock-crusher** [reply]
chadhobson, If the electrical specs you state above are correct then it would suggest that your hat relays are sufficient.

**Reply by chadhobson** [reply]
I still don't understand how wiring the chlorinator and pump to the same source makes it any safer. I could still walk up and manually turn off the pump at the keypad and leave the chlorinator on and assuming it had a bad flow switch, it would still try to make chlorine and potentially go boom. It feels like security theater to me.

Similarly, if I wire the chlorinator into a relay on the Nixie, the controller can still turn it on even though the pump isn't flowing because the chlorinator has a bad flow switch. It seems like a bad flow switch foils whatever security you _think_ you're introducting.

Or I'm just totally not understanding the concept which is not outside the realm of possibility. Also, there's a "standalone mode" mentioned where you just run the pump and chlorinator and hook the 485 cable between the two and walk away, so Pentair doesn't seem to be as worried about pipe bombs.

**Reply by rstrouse** [reply]
That isn’t what makes it safer.  When the filter circuit is not engaged it will cut power so that the cell cannot generate.  It isn’t that they are on the same circuit it is that the power to the surge board is completely cut under nearly all potential failure modes.

**Reply by chadhobson** [reply]
I get that. The absence of power to the shared circuit means neither can turn on and vice versa. But still, and sorry if I'm being obtuse, how does that help in the event of a bad flow switch in the chlorinator which , to me, still seems to be the only scenario for exploding PVC?

Why the need for a controller relay on the chlorinator if both the chlorinator and pump draw power from the same hardwired circuit and both are on 24/7 unless the breaker is shut off (winter)? I'm starting to wonder if this may be tied to people in warmer weather climates who can run their pool year round vs those of us who can't. Like it somehow makes sense in your case but not mine, if that makes sense?

Bottom line... if both have power, then explosion is possible with bad flow switch in chlorinator. I still don't see anything that prevents that other than a secondary flow switch. What is the catalyst in this system (other than the chlorinator) that can detect a bad flow switch and shut it all down? I think thats the part I'm failing to grasp.

I really appreciate everyone's input, BTW. So thank you!

**Reply by chadhobson** [reply]
One other comment... when I say "bad flow switch", I'm thinking of one thats failing saying there IS flow when there is not.

**Reply by orrious** [reply]
> The fact that a filter pump is controlled via RS485 does not change the above statements. 

@rock-crusher You're right, RS485 to the pump doesn't have anything to do with the above statements, but that's not what I was talking about.  I asked if the Intellichlor is turned off via RS485 when the pump is turned off.

**Reply by rock-crusher** [reply]
`@chadhobson   I still don't understand how wiring the chlorinator and pump to the same source makes it any safer.`

Well, should the breaker trip, both the pump and SCG will shut down and be powerless, same if the breaker was turned off for service purposes.

`@chadhobson    I could still walk up and manually turn off the pump at the keypad and leave the chlorinator on and assuming it had a bad flow switch, it would still try to make chlorine and potentially go boom. It feels like security theater to me.`

If you are connected to Nixie, or one of the Pentair controllers and set-up properly the keypad on your pump becomes disabled, you can no longer control it from there.

`@chadhobson   Similarly, if I wire the chlorinator into a relay on the Nixie, the controller can still turn it on even though the pump isn't flowing because the chlorinator has a bad flow switch. It seems like a bad flow switch foils whatever security you think you're introducting.`

Not just any relay in Nixie, you must connect to the designated filter pump relay. Then, when activated, both the filter pump via RS485 and the SCG will respond to the on/off command. They work in unison, on together and off together.

`@chadhobson   Also, there's a "standalone mode" mentioned where you just run the pump and chlorinator and hook the 485 cable between the two and walk away, so Pentair doesn't seem to be as worried about pipe bombs.`

Not so fast, you missed the TECHNICAL DATA which states:

“ Note: A TIME CLOCK must be used if the iChlor SCG is used in “stand alone” mode without an automation control system”

Look, I don’t make this sh*t up, I’m just sharing the little info I’ve learned. 

If you’re using Nixie for automation this added safety feature from the manufacture is simple to employ into the project, so, why not? 

It’s a good idea, and good peace of mind.

You can choose to heed to it, or not.

Bottom line, although explosions are far and few between they do and have happened. I think a lot has to do with misunderstanding, incompetence, lack of maintenance and possibly improper installations.

`@orrious    I asked if the Intellichlor is turned off via RS485 when the pump is turned off.`

Sorry, as you, I only know what I’ve learned from reading the manual, so I don’t have an answer for you on that specifically. There seems to be an internal flow switch that will stop generation though.

What I can tell you is the SCG is always receiving power unless the power center is disconnected.

**Reply by orrious** [reply]
@rock-crusher I understand where you are coming from by wanting to adhere to the manual, however I prefer to learn about real world experience from other folks as well.  For instance, I do not have a pump relay.  My pump is powered 100% of the time and is directly connected to a dedicated breaker and is solely controlled by njs-PC via RS485, so there is no pump relay to connect my IC40 to.  I just finished wiring the IC40 RS485 and noticed that njs-PC does turn off the IC40 via RS485.

What I am attempting to solicit is other folks actual experience with their SWGs.  Is controlling via RS485 sufficient or has someone actually experienced an issue with their SWG not shutting down either due to flow or RS485 commands failing and would 100% recommend switching the mains to the IC40 and not rely on the RS485 commands.

**Reply by rstrouse** [reply]
The behavior of the Chlorinator is that it will stop responding to RS485 commands when the failure mode is open on the flow/temp switch.  However, if your njsPC psu dies or the pi takes a dump, the Chlorinator will simply switch to manual mode and continue to generate.  In this condition the hope is that the flow switch disengages the cell.  The flow switch design can allow it to fail on because corrosion can form at the top of the spring paddle and can get stuck on.

**Reply by orrious** [reply]
Playing devil's advocate:  If the Pi PSU fails, the Chlorinator relay should fail open; so that solves that.  

In the case where the pi looses RS485 to the Chlorinator, even on a relay the Chlorinator will switch to manual mode and continue to generate.  What would trigger nodejs-poolcontroller to turn off the Chlorinator relay?  Maybe leverage the flow switch on the ORP/PH housing.  If there is no flow on that, then nodejs-poolcontroller would need to have a dedicated Chlorinator relay and should shut it off?

**Reply by chadhobson** [reply]
That was what I was getting at. A bad flow switch defeats whatever security you thought you had in place unless you have a backup flow switch or...

One of the items I went ahead and bought off Amazon was a liquid pressure sensor I saw someone else using on their filter in another discussion here. I don't know yet how much automation possibility exists in this app because I don't have it reading anything yet but that could be a good failover.

If no pressure is detected at the filter, shut off the chlorinator. Now we have a backup sensor if the chlorinator flow switch fails.

**Reply by chadhobson** [reply]
_If you are connected to Nixie, or one of the Pentair controllers and set-up properly the keypad on your pump becomes disabled, you can no longer control it from there._

@rock-crusher Thanks for this tidbit. I had no idea which is weird because every YouTube video I watched on these pumps before I bought this one shows people using the keypads on them to turn speeds up and down and showing the GPM difference. And most of these are VERY nice pools that have to have a controller, so if the keypad gets turned off, is there some kind of override these guys are using?

**Reply by rock-crusher** [reply]
`@chadhobson   so if the keypad gets turned off, is there some kind of override these guys are using?` 

Service Mode in the OCP

**Reply by rock-crusher** [reply]
`@orrious   and would 100% recommend switching the mains to the IC40 and not rely on the RS485 commands.` 

Lol… Just thought I’d throw this bit of info into the mix regarding the Pentair SCG wiring instructions, or any other for that matter.

![Nec Section 110 3 (B)](https://user-images.githubusercontent.com/94593954/174417103-24833aad-896a-43ba-bf54-344ad1e6e592.jpg)

Well, looks as though the National Electrical Code and Osha both require it 100%

**Comment by chadhobson** [comment]
Switching gears to something to more pressing for me right now...

As I said before, I'm redoing all the plumbing on the pad and I'd like to future-proof as much as possible. In particular, I'm looking for ORP/PH sensors later. I've currently got a Pentair water temp sensor and a pressure sensor for the filter. So..

1. Any other sensors I'm not aware of that you guys and gals are using? I'd love a GPM sensor  instead of a gauge but I'm not having any luck finding anything priced in reality.
2. I'd like to pre-plumb for the ORP/PH sensors, so can anyone share a pic of their sensors in place with the surrounding plumbing? I'm trying to figure out where they go in the series of already existing equipment and sensors.

Thanks!

**Reply by johnny2678** [reply]
1. Maybe a flow sensor for the chem flow cell if yours didn't come with one?  I've got a few ds18b20s tapped in that I use for solar temp monitoring, but hey are hardly necessary. I'm with you on the GPM sensor but haven't found anything reasonable for 2" pvc
2. flow cell inlet line should be just after your filter and flow cell outlet can go into the pump suction port.  Good diagram [here](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-REM-Chem-Controller#plumb-the-flow-cell):

![image](https://user-images.githubusercontent.com/8219743/174462162-97c33e9f-594d-43fe-bb48-b985b434523e.png)

Can post some pics of mine tmr when it's light out.

**Reply by chadhobson** [reply]
First big pool party of the year is behind me, so I can now move on to cutting PVC and the inherent downtime that will, no doubt, result. :-(

I have a couple questions at this point that I'm still unclear on.

1. My current PoolPilot is bonded with the rest of my equipment. It's also a non-metal case, so where are you all attaching your bond wire to for this new case? The lug on the outside, bottom left of the chassis?

2. Pentair wants a loop for the iChlor if the GPM is above 85 GPM. I have no idea if this new pump is or is not (I suspect it is at higher speeds based on filter pressure vs my old pump) but the AutoPilot has the same setup so I'm leaning towards mimicking what I have now. Is that the right approach? Any difference with vertical vs horizontal mounting of the iChlor? You can see a pic of my setup here, from before I started down this path...

![IMG_3023](https://user-images.githubusercontent.com/1839161/176554127-c2614689-6c0c-4632-a042-e77b1800b3ab.jpeg)

**Reply by tagyoureit** [reply]
1. Yes, the outside nut is the bonding lug.
2. If your setup was working, you are safe to replicate it.  I found this: `For this reason, Pentair introduced the i1 Intelliflo which is restricted to 65gpm at 60 feet total dynamic feet of head, even if it is set at the maximum 3450 RPM setting.` from [Pool Supply Unlimited](https://www.poolsupplyunlimited.com/pool/pentair-ec-011059-intelliflo-variable-speed-pump/160346p1) but haven't seen that same restriction explicitly stated elsewhere.

**Reply by chadhobson** [reply]
@tagyoureit, Interesting. Here's the chart from Pentair. Looks like even at 3450 RPM its still under 85GPM. I haven't found a reason to run this pump higher than 2850 RPM as its just moving a TON of water even at that speed vs 3450 RPM on my old single speed. The jet on the steps comes out so hard at 2850 RPM it hurts your leg if it's next to it.

![Pentair i1 Flow Rate](https://user-images.githubusercontent.com/1839161/176562212-422d77b9-0b15-4ac5-923f-c6dd81d14968.png)

So if this pump isn't capable of doing more than 85 GPM, maybe I should ditch the loop? I'm trying to visualize what that looks like on my current pad. I kind of like how the chlorinator is up off the ground making it easier to see so the landscapers don't step on it (like they do the pipes laying on the ground that I have to repair every few years because they break them).

I was thinking about routing the plumbing right off the heater, back towards the fence, and put the chlorinator there so its 100% out of the way and easy to access. Thoughts?

**Reply by chadhobson** [reply]
Actually, I have my axis reversed when reading that chart. I don't know the head or how to calculate it. But it's an in-ground pool and the pump is at ground level. Highest point of water at the pump is roughly 3 feet above the water line of the pool. Then there's a max 20 feet of pipe to the pool and back (not sure if you use both for total pipe length or just one side) for both the skimmer and the main drain.

How do you calculate the head if you have no clue what the friction loss is on all the pipes and fittings?

**Reply by tagyoureit** [reply]
I have no idea how to calculate any of that nor do I know (or care) about the ratings for my pool.  Rule of thumb would be the shortest amount of rise and fewest number of turns will result in the least amount of friction through the system.  I know vertical runs will add some but not nearly as much as the former two.  

I also saw those charts and couldn't correlate it with the statement I pasted above - hence the vote of 'no confidence' along with the statement.  

Personally, I have a VS pump that goes up to 3450 rpm and I don't have a chlorinator bypass (previously IC-40 and now an IC-60).  Given that they are $1300++ now I'd opt for the bypass and spend a few extra bucks on PVC to try and not push the limits of the chlor in hopes that it lasts a few extra years.

If you really want to go down the rathole, from the Pentair intellichlor manual:
```
The IntelliChlor® SCG is designed to operate with water flow rates from 25 +/- 5 gallons per minute (gpm) 
up to 105 gpm. Refer to the pipe manufacturer specifications for recommended maximum flow rate. For 
flow rates over 80 gpm you must use a bypass loop (as shown below) for best chlorine production.  
Installations with flow rates over 80 gpm include those that have in-floor cleaning systems or booster 
pumps. These systems MUST use a bypass loop with the IntelliChlor SCG with a flow control valve that
assures that the flow through the IntelliChlor SCG is maintained within it's designed operating water 
flow rates.
```
Now why would they build a piece of equipment rated to 105gpm but then effectively cap it at 80gpm?
